composer.json

```diff
@@ -6,60 +6,71 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.0",
         "cweagans/composer-patches": "~1.4",
-        "drupal/core": "^9.3",
+        "drupal/core": "^9.4",
         "drupal/smart_trim": "^1.2",
         "drupal/twig_field_value": "^2.0",
-        "drupal/ui_patterns": "^1.0",
+        "drupal/ui_patterns": "^1.4",
         "openeuropa/ecl-twig-loader": "^3.1",
-        "openeuropa/oe_time_caching": "~1.0"
+        "openeuropa/oe_time_caching": "dev-master"
     },
     "require-dev": {
         "behat/behat": "^3.10",
-        "behat/mink-extension": "^2.3.1",
+        "behat/mink-selenium2-driver": "^1.5",
         "composer/installers": "~1.5",
-        "drupal/address": "~1.9",
+        "doctrine/collections": "^1.8",
```

Comment:
New explicit dependency wasn't mentioned in `_readme`.

Reply:
Fixed.